### ISSUE TITLE
Parallel debugging description

### ISSUE THREAD

**Issue by dongahn** [flux-framework/flux-docs#47]
As discussed this at 11AM meeting 7/16, we need a high level text on how to parallel debug a flux job. A good time will be after Perforce Software's support engineer will have a chance to poke at a Flux version on one of LC TOSS clusters. I will coordinate. 

In the meanwhile, should this text go into https://flux-framework.readthedocs.io/en/latest/quickstart.html or somewhere else?

**Comment by cmoussa1** [comment]
> In the meanwhile, should this text go into https://flux-framework.readthedocs.io/en/latest/quickstart.html or somewhere else?

Yeah, I think it seems reasonable to have a parallel debugging section on the Quickstart page for now. Eventually, maybe we can piece that section out and have a separate page just on debugging?

**Comment by dongahn** [comment]
> Eventually, maybe we can piece that section out and have a separate page just on debugging?

Debugging and performance tuning, and yes eventually :-)

**Comment by dongahn** [comment]
https://github.com/flux-framework/flux-sched/pull/694#issuecomment-663890636

I will be seeking some testing from Perforce and document in the next few weeks. 

As part of this testing, some more best-practice recommendations as to how to configure the TotalView parallel debugger will be likely come out. That should be documented too.
